bin/pycbc_merge_inj_hdf

```diff
@@ -65,7 +65,7 @@ if __name__ == '__main__':
         injections = pycbc.inject.InjectionSet(inj_file)
 
         inj_table = injections.table
-        inj_table = inj_table[np.sort(inj_table.dtype.fields.keys())]
+        inj_table = inj_table[np.sort(list(inj_table.dtype.fields.keys()))]
```

Comment:
It's probably a good idea to put in a comment saying why this sort is needed (which I am not quite sure of).